applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -526,24 +526,103 @@ Use the following configuration options to customize the number of supported bri
+            You can increase the number of Bluetooth LE connections if you decrease the size of the Bluetooth LE TX/RX buffers used by the Bluetooth controller, but this will decrease the communication throughput.
+            Build the target using the following command in the project directory to enable a configuration that increases the number of Bluetooth LE connections to ``20`` by decreasing the sizes of Bluetooth LE TX/RX buffers:
+
+            .. parsed-literal::
+               :class: highlight
+
+               west build -b nrf5340dk/nrf5340/cpuapp -p -- -Dmatter_bridge_SHIELD=nrf7002ek -DSB_CONFIG_WIFI_PATCHES_EXT_FLASH_STORE=y -DSB_CONFIG_DFU_MULTI_IMAGE_PACKAGE_WIFI_FW_PATCH=y -DSB_CONFIG_WIFI_NRF700X=y -Dmcuboot_CONFIG_UPDATEABLE_IMAGE_NUMBER=3 -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf"
+
+
+         .. group-tab:: Matter-Bridge over thread configuration
+
+            Increasing the number of Bluetooth LE connections affects the RAM usage on both the application and network cores.
+            The current maximum number of Bluetooth LE connections that can be selected using the default configuration is ``10``.
+            Due to using thread and BLE protocols in this configuration You can't increase the number of Bluetooth LE connections and additionally you have to decrease the size of the Bluetooth LE TX/RX buffers used by the Bluetooth controller, this will decrease the communication throughput.
+            Build the target using the following command in the project directory to enable a configuration that allow to set the number of Bluetooth LE connections to ``10`` by decreasing the sizes of Bluetooth LE TX/RX buffers:
+
+            .. parsed-literal::
+               :class: highlight
+
+               west build -b nrf5340dk/nrf5340/cpuapp -p -- -DCONFIG_BRIDGED_DEVICE_BT=y -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net_thread_nrf5340.conf" -DEXTRA_CONF_FILE="overlay-bt_max_connections_app_thread_nrf5340.conf"
+
+   .. group-tab:: nRF54 DKs
+
+      You have to consider several factors to decide how many Bluetooth LE bridged devices to support.
+      Every Bluetooth LE bridged device uses a separate Bluetooth LE connection, so you need to consider the number of supported Bluetooth LE connections (selected using the :kconfig:option:`CONFIG_BT_MAX_CONN` Kconfig option) when you configure the number of devices.
+      Since the Matter stack uses one Bluetooth LE connection for commissioning, the maximum number of connections you can use for bridged devices is one less than is configured using the :kconfig:option:`CONFIG_BT_MAX_CONN` Kconfig option.
+
+      .. tabs::
+
+         .. group-tab:: Matter-Bridge over wifi configuration
+
+            Increasing the number of Bluetooth LE connections affects the RAM usage on both the application and network cores.
```

Comment:
I can see a lot of duplications in this text between the tabs. I would suggest contacting @wiba-nordic to discuss how to avoid it, as the only things that changes are build command and the connections number.

Reply:
I'll take a look once the doc build finishes. Hopefully there are parts we can move to be outside of the tabs, and maybe some parts we can have as includes.